src/content/en/fundamentals/accessibility/00-intro/intro-three.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: shared/narrow
+title: "Understanding Users' Diversity"
+description: "Accessibility Codelab"
+published_on: 2016-03-01
+updated_on: 2016-03-01
+order: 3
+translation_priority: 0
+authors:
+  - megginkearney
+  - dgash
+key-takeaways:
+  tldr: 
+    - "Learn what accessibility means and how it applies to web development."
+    - "Learn how to make web sites accessible and usable for everyone."
+    - "Learn how to include basic accessibility with minimal development impace."
+    - "Learn what HTML features are available and how to use them to improve accessibility."
+    - "Learn about advanced accessibility techniques for creating polished accessibility experiences."
+notes:
+  efficiency:
+    - "Understanding the accessibility issue, its scope, and its impact can make you a better web developer."
+  problem-solving:
+    - "Catching, identifying, and removing potential accessibility roadblocks before they happen can improve your development process and reduce maintenance requirements."
+---
+
+To explain further, here's an informative question/answer session with Victor Tsaran, a Technical Program Manager at Google, who is totally blind.
```

Comment:
I think someone just landing on this page won't have any context. Maybe it could lead off with something like "When learning about accessibility it helps to have an understanding of the diverse range of users in the world and the kinds of accessibility topics that effect them. To explain further..."